aptos-move/framework/src/natives/cryptography/algebra/arithmetics/mod.rs

```diff
@@ -12,13 +12,13 @@ pub mod sub;
 
 #[macro_export]
 macro_rules! ark_binary_op_internal {
-    ($context:expr, $args:ident, $ark_typ:ty, $ark_func:ident, $gas:expr) => {{
-        let handle_2 = safely_pop_arg!($args, u64) as usize;
-        let handle_1 = safely_pop_arg!($args, u64) as usize;
+    ($context:expr, $args:ident, $ark_typ:ty, $ark_func:expr, $gas:expr) => {{
+        let handle_2 = aptos_native_interface::safely_pop_arg!($args, u64) as usize;
+        let handle_1 = aptos_native_interface::safely_pop_arg!($args, u64) as usize;
         safe_borrow_element!($context, handle_1, $ark_typ, element_1_ptr, element_1);
         safe_borrow_element!($context, handle_2, $ark_typ, element_2_ptr, element_2);
         $context.charge($gas)?;
-        let new_element = element_1.$ark_func(element_2);
+        let new_element = $ark_func(element_1, element_2);
```

Comment:
Was there a problem before, or why this change?

Reply:
Hm. Strange. Looping in @zjma since he should be made aware of this.

Reply:
I tried the old macro and it builds and tests fine...
@vgao1996 here's a revert sub-PR: https://github.com/vgao1996/aptos-core/pull/2/files

Reply:
Let me double check

Reply:
Okay, looks like it's a rust-analyzer problem. I've reverted these changes.